### ISSUE TITLE
Focus is lost after closing the Tabs overflow with ESC key

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3509]
## Describe the bug

When tabs overflow is open, hitting ESC causes focus to be lost to the page.

## How to reproduce

- Go to https://v1.clarity.design/tabs
- Using the keyboard alone, activate the overflow feature in the Tab Overflow example
- Press the Escape key to close the overflow feature
- Try to determine which element has keyboard focus

Focus is lost after closing the Tabs overflow with the Escape key

## Expected behavior

Focus should be set to the button that trigged the appearance of the tab overflow. 

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by Jinnie** [comment]
This will be handled by https://github.com/vmware/clarity/pull/3609
That was the reason it was assigned to me.
Please contact assignee before unassigning assigned issues.

**Comment by gnomeontherun** [comment]
@Jinnie this is done then right?

**Comment by Jinnie** [comment]
Yes, it's handled by the current implementation.
